### PR TITLE
Document `outletName` constraint

### PR DESCRIPTION
**Pull request by knownasilya**
The long needed docs for https://github.com/ember-animation/liquid-fire/pull/341

### CODE REVIEW

File: tests/dummy/app/templates/transition-map/outlet-constraints.hbs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<h2>Outlet Constraints</h2>
+
+<p>
+  The following three constraints apply to oulet names.
```
Comment (ef4) [inline tests/dummy/app/templates/transition-map/outlet-constraints.hbs]: Copy paste error: not "three".

Comment (knownasilya) [inline tests/dummy/app/templates/transition-map/outlet-constraints.hbs]: Thanks!

**Comment by knownasilya** [comment]
@ef4 should be set. Not sure why travis died.

**Comment by ef4** [comment]
Thanks.

**Comment by knownasilya** [comment]
Any plans to deploy the docs for this change?

**Pull request merged**